variables.tf

```diff
@@ -70,31 +73,52 @@ variable "members" {
   #   "alice"
   # ]
   #
+
   default = []
 }
 
 variable "admin_repositories" {
-  description = "A list of repository names the current team should get admin (full) permission to."
+  description = "(Optional) A list of repository names the current team should get admin (full) permission to."
   type        = set(string)
+
   #
   # Example:
   #
-  # admin_reporitores = [
+  # admin_repositories = [
   #   "bob",
   #   "alice"
   # ]
   #
+
   default = []
 }
 
 variable "push_repositories" {
-  description = "A list of repository names the current team should get push (read-write) permission to."
+  description = "(Optional) A list of repository names the current team should get push (read-write) permission to."
   type        = set(string)
   default     = []
 }
 
 variable "pull_repositories" {
-  description = "A list of repository names the current team should get pull (read-only) permission to."
+  description = "(Optional) A list of repository names the current team should get pull (read-only) permission to."
   type        = set(string)
   default     = []
 }
+
+# ------------------------------------------------------------------------------
+# MODULE CONFIGURATION PARAMETERS
+# These variables are used to configure the module.
+# See https://medium.com/mineiros/the-ultimate-guide-on-how-to-write-terraform-modules-part-1-81f86d31f024
+# ------------------------------------------------------------------------------
+
+variable "module_enabled" {
+  type        = bool
+  description = "(Optional) Whether to create resources within the module or not. Default is true."
+  default     = true
+}
```

Comment:
this will break existing installations.. there is no benefit in breaking this behavior.

Reply:
removed again